modules/search/instant-search/components/search-results.jsx

```diff
@@ -58,17 +62,33 @@ class SearchResults extends Component {
 			>
 				<SearchForm className="jetpack-instant-search__search-results-search-form" />
 
-				{ hasResults && (
-					<p className="jetpack-instant-search__search-results-real-query">
-						{ this.getSearchTitle() }
-					</p>
-				) }
+				<div
+					className={
+						hasResults
+							? 'jetpack-instant-search__search-results-real-query'
+							: 'etpack-instant-search__search-results-empty'
+					}
+				>
+					{ this.getSearchTitle() }
+				</div>
+
 				{ hasResults && hasCorrectedQuery && (
 					<p className="jetpack-instant-search__search-results-unused-query">
 						{ sprintf( __( 'No results for "%s"', 'jetpack' ), query ) }
 					</p>
 				) }
-				{ hasResults && (
+				{ this.props.hasError && (
+					<Notice type="warning">
```

Comment:
How about we add a button to these notices that just says "Retry" and then shorten this text to just say "It looks like you are offline."